modules/fastqc/functions.nf

```diff
@@ -37,7 +44,9 @@ def getPathFromList(path_list) {
 // Function to save/publish module results
 //
 def saveFiles(Map args) {
-    if (!args.filename.endsWith('.version.txt')) {
+    // TODO better way to detect that we are in a test and want to emit `versions.yml`
+    // or a least use a dedicated, unique environment variable.
+    if (!args.filename.equals('versions.yml') || System.getenv("PROFILE")) {
```

Comment:
I added a custom pytest-workflow test to ensure `versions.yml` is valid. 
The problem is that `versions.yml` needs to be published to be accessible from the test. By default, we exclude `verions.yml` from `publishDir`. This is my current workaround, maybe someone has a better idea...

Reply:
@nf-core/modules-team, any suggestions how to solve this cleanly?

Reply:
Could we maybe export a custom variable just for testing with nf-core/modules? or check if `pytest` is available as a command? 🤔

Reply:
we decided to go for a variable, see discussion on slack: 
https://nfcore.slack.com/archives/CJRH30T6V/p1632300389380200